examples/inverse/plot_morph_VolSourceEstimate.py

```diff
@@ -0,0 +1,282 @@
+                        noise_cov=noise_cov, pick_ori='max-power',
+                        weight_norm='nai')
+
+    # Apply this spatial filter to the evoked data.
+    stc = apply_lcmv(evoked, filters, max_ori_out='signed')
+
+    # take absolute values for plotting
+    stc.data[:, :] = np.abs(stc.data)
+
+    # Save result in stc files if desired
+    if fname is not None:
+        stc.save('lcmv-vol')
+
+    # select time window (tmin, tmax) in ms - consider changing for real data
+    # scenario, since those values were chosen to optimize computation time
+    stc.crop(0.087, 0.087)
+
+    src = forward['src']
+
+    # Save result in a 4D nifti file
+    img = mne.save_stc_as_volume(fname, stc, src,
+                                 mri_resolution=True)
+
+    return img, stc, src
+
+
+###############################################################################
+# Save non linear mapping data
+
+def write_morph(fname, morph, overwrite=True):
+    morph_out = dict()
+
+    # dissolve object structure
+    for key, value in morph.items():
+        # save type for order independent decomposition
+        if hasattr(value, '__dict__'):
+            value = value.__dict__
+        morph_out[key] = value
+    if not fname.endswith('.h5'):
+        fname += '.h5'
+    write_hdf5(fname, morph_out, overwrite=overwrite)
+
+
+###############################################################################
+# Load non linear mapping data
+
+def read_morph(fname):
+    if not fname.endswith('.h5'):
+        fname += '.h5'
+    morph_in = read_hdf5(fname)
+
+    morph = dict()
+    # create new instances
+    morph['mapping'] = imwarp.DiffeomorphicMap(None, [])
+    morph['mapping'].__dict__ = morph_in.get('mapping')
+    morph['affine'] = imaffine.AffineMap(None)
+    morph['affine'].__dict__ = morph_in.get('affine')
+
+    morph['affine_reg'] = morph_in.get('affine_reg')
+    morph['domain_shape'] = morph_in.get('domain_shape')
+
+    return morph
+
+
+###############################################################################
+# Execute example
+
+# Settings
+voxel_size = (5., 5., 5.)  # of the destination volume
+
+# Setup path
+data_path = sample.data_path()
+
+t1_sample = data_path + '/subjects/sample/mri/brain.mgz'
+t1_fsaverage = data_path + '/subjects/fsaverage/mri/brain.mgz'
+
+# compute LCMV beamformer inverse example
+img, stc, src = compute_lcmv_example_data(data_path)
+
+###############################################################################
+# Compute Morph Matrix and Dispersion Volume
+morph = mne.source_estimate.compute_morph_sdr(
+    t1_sample,
+    t1_fsaverage,
+    voxel_size=voxel_size)
+
+write_morph('SDR-sample-fsaverage', morph)
+
+###############################################################################
+# Morph Pre - Computed
+
+morph_precomputed = read_morph('SDR-sample-fsaverage')
+
+stc_vol_to_preC = stc.morph_precomputed('sample', 'fsaverage', src,
```

Comment:
don't use camelCase vars in var names (just CamelCase for class defs). I'd probably call this `stc_vol_morphed` or `stc_vol_fsaverage`

Reply:
check